django/db/models/expressions.py

```diff
@@ -26,11 +27,9 @@ class SQLiteNumericMixin:
 
     def as_sqlite(self, compiler, connection, **extra_context):
         sql, params = self.as_sql(compiler, connection, **extra_context)
-        try:
+        with contextlib.suppress(FieldError):
```

Comment:
This was already discussed and reverted, see 6e4c6281dbb7ee12bcdc22620894edb4e9cf623f.